src/main/java/edu/cornell/kfs/vnd/batch/service/impl/VendorEmployeeComparisonReportServiceImpl.java

```diff
@@ -0,0 +1,171 @@
+package edu.cornell.kfs.vnd.batch.service.impl;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.function.Predicate;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.core.api.config.property.ConfigurationService;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.KFSConstants.OptionLabels;
+
+import edu.cornell.kfs.sys.batch.CuBatchFileUtils;
+import edu.cornell.kfs.sys.service.ReportWriterService;
+import edu.cornell.kfs.sys.util.ForUnitTestConvenience;
+import edu.cornell.kfs.vnd.CUVendorKeyConstants.EmployeeComparisonReportKeys;
+import edu.cornell.kfs.vnd.batch.VendorEmployeeComparisonResultCsv;
+import edu.cornell.kfs.vnd.batch.service.VendorEmployeeComparisonReportService;
+import edu.cornell.kfs.vnd.businessobject.VendorEmployeeComparisonResult;
+
+public class VendorEmployeeComparisonReportServiceImpl implements VendorEmployeeComparisonReportService {
+
+    private enum ReportStatistic {
+        TOTAL_EMPLOYEES(resultRow -> true),
+        TOTAL_ACTIVE_EMPLOYEES(resultRow -> resultRow.isActive()),
```

Comment:
The expressions in the enum's other constants are set up as lambdas since they don't have a straightforward way of being defined as method references (aside from adding methods that may introduce too much clutter). I was trying to keep these as lambdas for consistency, but if you think a method reference would still be better, I could change this accordingly.

Reply:
It looks like simplifying this one to a method reference is feasible, so I'll proceed with that.